core/platform_feature_list.py

```diff
@@ -18,14 +18,40 @@
 
 from __future__ import annotations
 
+import enum
+
 from core.domain import platform_parameter_list as params
 
 from typing import List
 
-ParamNames = params.ParamNames
 
-# Names of feature objects defined in domain/platform_parameter_list.py
-# should be added to one of the following lists:
+class FeatureNames(enum.Enum):
```

Comment:
Which backend test verifies that this matches other lists of features we have? (We must avoid skew.)

Reply:
We do not have any other lists for features, this is the only source of truth. We do have several tests present in the relevant test file, like all the feature present in the enums actually exists, no duplicates are present, etc.

Reply:
Shouldn't platform_parameter_list be updated to remove these then? And extending @seanlip's thought, perhaps we need a test to verify that the two lists are mutually exclusive (since features should go here and other configuration values should go in platform_parameter_list).

Reply:
Hey @BenHenning, I have created an issue #19285 to perform things after this PR gets in. I'm a bit confused about the second para. We plan to separate the platform_fature_list and platform_parameter_list, I'm a bit confused on what we are referring to when you mention "configuration values".
Thanks

Reply:
I think @BenHenning just wants to ensure there's a backend test to verify that the feature flag names and platform parameter names are distinct, to avoid any confusion.

Reply:
Thanks @seanlip and @BenHenning, I have updated the #19285 issue to perform this task. I will perform this in the subsequent PR because.

Reply:
@lkbhitesh07 I think your previous sentence is truncated -- because what?

Reply:
Oh!
Actually, as the feature flags were part of the platform parameters and this PR only separates the feature flags and platform parameters and do not touch the parameters code, there might be feature flags names that are part of platform parameters, they are not getting used. I will be doing the cleanup just after this PR.
Thanks